lib/webview-helpers.js

```diff
@@ -0,0 +1,111 @@
+import _ from 'lodash';
+import logger from './logger';
+import { asyncMap } from 'asyncbox';
+
+const NATIVE_WIN = "NATIVE_APP";
+const WEBVIEW_WIN = "WEBVIEW";
+const WEBVIEW_BASE = WEBVIEW_WIN + "_";
+const CHROMIUM_WIN = "CHROMIUM";
+
+let helpers = {};
+
+// This function gets a list of android system processes and returns ones
+// that look like webviews, with the appropriate webview prefix and their PID
+async function webviewsFromProcs (adb, deviceSocket) {
```

Comment:
It might just be me, but this function, and `procFromWebview`, seem like they ought to be part of `appium-adb`. They seem to expose internals to the Android OS which are out of place here.